### ISSUE TITLE
mpileup: invalid option -- 't'

### ISSUE THREAD

**Issue by Mx-Chpl** [sanger-pathogens/ariba#326]
Hello,
I want to use the last version of ARIBA (XXX) but an error occured when I ran ariba test or directly my samples.
Here all the error messages (with ariba test) :
_ Assembling each cluster ___________________________
Will run 1 cluster(s) in parallel
Not constructing cluster cluster because it only has 2 reads (1 of 9)
Constructing cluster cluster_1 (2 of 9)
Constructing cluster cluster_2 (3 of 9)
Constructing cluster cluster_3 (4 of 9)
Constructing cluster cluster_4 (5 of 9)
Constructing cluster gene (6 of 9)
Constructing cluster gene_1 (7 of 9)
Constructing cluster noncoding (8 of 9)
Constructing cluster noncoding_1 (9 of 9)
Start running cluster cluster_1 in directory /home/chapelm/Ariba_test/OUT/ariba.tmp.1fuiuzdn/cluster_1
cluster_1 detected 1 threads available to it
mpileup: invalid option -- 't'
Failed cluster: cluster_1
Finished running cluster cluster_1 in directory /home/chapelm/Ariba_test/OUT/ariba.tmp.1fuiuzdn/cluster_1
Deleting cluster dir /home/chapelm/Ariba_test/OUT/ariba.tmp.1fuiuzdn/cluster_1
Other clusters failed. Will not start cluster cluster_2
Other clusters failed. Will not start cluster cluster_3
Other clusters failed. Will not start cluster cluster_4
Other clusters failed. Will not start cluster gene
Other clusters failed. Will not start cluster gene_1
Other clusters failed. Will not start cluster noncoding
Other clusters failed. Will not start cluster noncoding_1
Final value of remaining_clusters counter: Value('l', 7)
Finished assembling clusters

Traceback (most recent call last):
  File "/Softs/virtualenvPython/Ariba_2.14.6/lib64/python3.6/site-packages/ariba/clusters.py", line 612, in run
    self._run()
  File "/Softs/virtualenvPython/Ariba_2.14.6/lib64/python3.6/site-packages/ariba/clusters.py", line 646, in _run
    raise Error('At least one cluster failed! Stopping...')
ariba.clusters.Error: At least one cluster failed! Stopping...

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Softs/virtualenvPython/Ariba_2.14.6/bin/ariba", line 312, in <module>
    args.func(args)
  File "/Softs/virtualenvPython/Ariba_2.14.6/lib64/python3.6/site-packages/ariba/tasks/run.py", line 65, in run
    c.run()
  File "/Softs/virtualenvPython/Ariba_2.14.6/lib64/python3.6/site-packages/ariba/clusters.py", line 615, in run
    raise Error('Something went wrong during ariba run. Cannot continue. Error was:\n' + str(err))
ariba.clusters.Error: Something went wrong during ariba run. Cannot continue. Error was:
At least one cluster failed! Stopping...

Something went wrong. See above for error message(s). Return code was 1

It seems to be an error with mpileup but I am not sure.

**Comment by emollier** [comment]
This looks to be a duplicate of #327 and can probably be closed now that it is properly addressed.
Have a nice day,  :)
Étienne.

**Comment by martinghunt** [comment]
Thank you! Yes, is duplicate
